src/Resources/config/services/form.xml

```diff
@@ -39,16 +39,16 @@
         </service>
 
         <service id="sylius.form.type.checkout_address" class="Sylius\Bundle\CoreBundle\Form\Type\Checkout\AddressType">
+            <argument type="service" id="sylius.address_comparator" />
```

Comment:
this service will probably be changed in sylius 2.0.0 to sylius.comparator.address